app/controllers/crosswalks_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CrosswalksController < ApplicationController
```

Comment:
Shouldn't this (and the corresponding views) be `crosswalk_issues` instead of `crosswalks`?

Reply:
The list displayed is of crosswalk_issues, but the plan for this page is to allow the end user to create crosswalk records, which will be the eventual replacement of the crosswalk CSV (which also resolves the crosswalk_issues).